install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,239 @@
+[[install-config-configuring-nsx-t-sdn]]
+[%hardbreaks]
+= Configuring NSX-T SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[nsx-t-sdn-and-openshift]]
+== NSX-T SDN and {product-title}
+
+VMware NSX-T provides advanced software-defined networking (SDN), security, and visibility
+to container environments that simplifies IT operations and extends native {product-title} networking capabilities.
+
+NSX-T supports virtual machine, bare metal, and container workloads across multiple clusters. This allows
+organizations to have complete visibility using a single SDN across the entire environment.
+
+For more information on how NSX-T integrates with {product-title}, see the xref:../architecture/networking/network_plugins.adoc#nsx-sdn[NSX-T SDN in Available SDN plug-ins].
+
+[[nsx-t-sdn-operations-workflow]]
+== Example Topology
+
+One typical use case is to have a Tier-0 (T0) router that connects the physical system with the virtual environment and a Tier-1 (T1) router to act as a default gateway for the {product-title} VMs.
+
+Each VM has two vNICs: One vNIC connects to the Management Logical Switch for accessing the VMs. The other vNIC connects to a Dump Logical Switch and is used by `nsx-node-agent` to uplink the Pod networking. For further details, refer to the link:https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/nsxt_24_ncp_openshift.pdf[NSX Container Plug-in for OpenShift].
+
+The LoadBalancer used for configuring {product-title} Routes and all project T1 routers and Logical Switches are created automatically during the {product-title} installation. 
```

Comment:
s/LoadBalancer/load balancer

Reply:
@ahardin-rh Is this a kubernetes object, here? They use LoadBalancer.